src/main/java/org/gradle/playframework/tools/internal/routes/DefaultRoutesPostProcessor.java

```diff
@@ -12,6 +12,10 @@
 import java.nio.file.Path;
 import java.util.stream.Stream;
 
+// This post processor fixes build / project dependent comments (DATE and SOURCE) from the RoutesCompiler generated files:
+// @GENERATOR:play-routes-compiler
+// @(DATE): Mon Apr 03 10:27:51 CEST 2023
+// @(SOURCE):/private/var/folders/79/xmc9yr493y75ptry2_nrx3r00000gn/T/junit4995996226044083355/conf/routes
```

Comment:
nit: multi-line comments usually use the `/**` syntax.
Is 16-18 an example?

Reply:
Yes

Reply:
I adjusted the Javadoc comments 👍